src/Tests/View/InstallQuickInput.test.ts

```diff
@@ -64,6 +64,14 @@ suite('View', function() {
           quickInput.getToolchainEnv();
         });
       });
+
+      test('NEG: throw error when error is not undefined - getToolchainEnv', function() {
```

Comment:
Since `error` is defined... how about following? :)
```suggestion
      test('NEG: throw error when toolchainEnv is not undefined, but error defined', function() {
```

Reply:
That's cool. I'll update this pr. Thank you!